src/elements/collapse.svelte

```diff
@@ -9,6 +9,7 @@ export let title = '';
 export let open = false;
 
 let root: HTMLElement;
+let variant = 'secretUI'
```

Comment:
Oh maybe change the name to something like `noBorderBackground` or something more descriptive

Reply:
changed